### ISSUE TITLE
NPE during Preconditions validation   

### ISSUE THREAD

**Issue by vgalloy** [GoogleContainerTools/jib#2702]
**Environment**:

- *Jib version:*  2.5.0
- *Build tool:* gradle 6.6
- *OS:* Ubuntu 20.04.1 LTS
- *Java:* openjdk version "14" 2020-03-17  

**Description of the issue**:
`ImmutableCollections#contains(Object)` throw a `NullPointerException` if the argument is `null`. Checking if collection contains `null` will always throw a `NullPointerException`
Example 
`Arrays.asList("java").contains(null)` => `false`
`List.of("java").contains(null)` => `NullPointerException`

There is several place where the following pattern is apply :
`Preconditions.checkArgument(!entrypoint.contains(null), "entrypoint contains null elements");`

Passing an ImmutableCollections as `entrypoint` always fail 

**Expected behavior**:
No `NullPointerException` exception

**Steps to reproduce**:
1. ./gradlew jibDockerBuild

**`jib-gradle-plugin` Configuration**: 
```groovy
plugins {
  id 'java-library'
  id 'com.google.cloud.tools.jib' version '2.5.0'
}

java {
  targetCompatibility = JavaVersion.VERSION_11
  sourceCompatibility = JavaVersion.VERSION_11
}

jib {
  container {
    entrypoint = List.of('java', '-jar', 'my-jar')
  }
}
```

**Log output**: 
```
Caused by: java.lang.NullPointerException
        at com.google.cloud.tools.jib.configuration.ContainerConfiguration$Builder.setEntrypoint(ContainerConfiguration.java:265)
        at com.google.cloud.tools.jib.api.JibContainerBuilder.setEntrypoint(JibContainerBuilder.java:274)
        at com.google.cloud.tools.jib.plugins.common.PluginConfigurationProcessor.processCommonConfiguration(PluginConfigurationProcessor.java:403)
        at com.google.cloud.tools.jib.plugins.common.PluginConfigurationProcessor.processCommonConfiguration(PluginConfigurationProcessor.java:455)
        at com.google.cloud.tools.jib.plugins.common.PluginConfigurationProcessor.createJibBuildRunnerForDockerDaemonImage(PluginConfigurationProcessor.java:132)
        at com.google.cloud.tools.jib.gradle.BuildDockerTask.buildDocker(BuildDockerTask.java:110)
```

**Additional Information**: 
I know that not a conventional way to write a `List`, I'm just trying to provided the shorter possible example.

**Comment by loosebazooka** [comment]
so the `contains` method cannot handle null on the specific implementation of `List`?

This is a good catch.

**Comment by loosebazooka** [comment]
@mpeddada1  can you look into this bug?

**Comment by mpeddada1** [comment]
@vgalloy We've released Jib 2.6.0, which contains a fix for this issue. Thanks for reporting!
